test_conformance/device_execution/enqueue_profiling.cpp

```diff
@@ -66,18 +66,18 @@ int test_enqueue_profiling(cl_device_id device, cl_context context,
     dev_queue = clCreateCommandQueueWithProperties(
         context, device, dev_queue_prop_def, &err_ret);
     test_error(err_ret,
-               "clCreateCommandQueueWithProperties(CL_QUEUE_DEVICE|CL_QUEUE_"
-               "DEFAULT) failed");
+               "clCreateCommandQueueWithProperties(CL_QUEUE_ON_DEVICE | "
+               "CL_QUEUE_ON_DEVICE_DEFAULT) failed");
```

Comment:
It might be helpful to add `CL_QUEUE_PROFILING_ENABLE` to this error message, also.